### PR TITLE
Updates Compose to alpha07

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -88,7 +94,10 @@ dependencies {
     implementation "androidx.compose.ui:ui-viewbinding:$rootProject.composeVersion"
     implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
     implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
+//    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
```
Comment (florina-muntenescu) [inline app/build.gradle:97]: should this be removed rather than commented out?

**Review by florina-muntenescu — APPROVED** [review]
Overall looks good. one extra question. feel free to merge after it's addressed

File: app/build.gradle (hunk 1 above)
Comment (chrisbanes) [reply app/build.gradle:97]: I'd keep it, we'll need to revert back to this for the actual release.

**Review by chrisbanes — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (manuelvicnt) [reply app/build.gradle:97]: +1, I left it commented out so that we don't forget to revert it

**Review by tiembo — APPROVED** [review]

**Pull request merged**